packages/scss/src/components/_actionIcon.scss

```diff
@@ -0,0 +1,20 @@
+.actionIcon {
```

Comment:
Multiple components use camelCase, inside a component, the - is used to separate parents and childs (e.g. .navSide-link)